share/docker/openfast_ubuntu/Dockerfile

```
@@ -13,42 +12,63 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+ARG BASE=ubuntu:jammy
 
-FROM ubuntu:bionic
+# Build stage 1: builds openfast.
+FROM ${BASE} AS build
 
 # Install dependencies
-
 # For gfortran-8
 # RUN add-apt-repository ppa:ubuntu-toolchain-r/test -y
 # apt-get install gfortran-8
 
-ENV DEBIAN_FRONTEND=noninteractive TZ=America/Denver
-
-RUN apt update -qq && \
-    apt install -y software-properties-common build-essential && \
-    add-apt-repository ppa:git-core/ppa -y && \
-    apt install -y python3-pip && \
-    apt install -y cmake cmake-curses-gui && \
-    apt install -y gcc gfortran make && \
-    apt install -y libblas-dev liblapack-dev && \
-    apt install -y git && \
-    apt install -y nano
+ENV DEBIAN_FRONTEND=noninteractive
 
-RUN pip3 install numpy
+RUN apt-get update -qq && \
+    apt-get install --no-install-recommends -y \
+        software-properties-common \
+        build-essential \
+        cmake  \
+        cmake-curses-gui \
+        gcc \
+        gfortran \
+        git \
+        libblas-dev \
+        liblapack-dev \
```

The OpenBLAS library has better performance than the native libraries in my experience.

```suggestion
        libopenblas-dev \
```

---

I'll update this now

---

Fixed by adding `libgomp1` to the final image!